Solutions/Endpoint Threat Protection Essentials/Hunting Queries/PersistViaIFEORegistryKey.yaml

```diff
@@ -61,10 +61,10 @@ query: |
   (
   imRegistry
   | where RegistryKey has_all ("HKEY_LOCAL_MACHINE", "Image File Execution Options")
-  | summarize Count=count() by Dvc, Username, RegistryKey
+  | summarize Count=count() by Dvc, RegistryKey
   | top 10 by Count desc
-  | extend AccountCustomEntity = Username, HostCustomEntity = Dvc
```

Comment:
The placement of the line was also correct. Did not get the intention of the change.
Please provide more details.

Reply:
We have KQL validation failing in this with the following error message: 
Template Id: f82c89fa-c969-4d12-832f-04d55d14522c is not valid in Line: 39 col: 36
Errors: The name 'Username' does not refer to any known column, table, variable or function., Code: 'KS142'